Code/Framework/AzCore/AzCore/Jobs/JobManagerComponent.cpp

```diff
@@ -46,9 +51,9 @@ namespace AZ
         JobManagerThreadDesc threadDesc;
 
         int numberOfWorkerThreads = m_numberOfWorkerThreads;
-        if (numberOfWorkerThreads <= 0)
+        if (numberOfWorkerThreads <= 0) // spawn default number of threads
         {
-            numberOfWorkerThreads = AZ::GetMin(static_cast<unsigned int>(desc.m_workerThreads.capacity()), AZStd::thread::hardware_concurrency());
+            numberOfWorkerThreads = AZ::GetMin(static_cast<unsigned int>(desc.m_workerThreads.capacity()), cl_numeratorJobThreads * AZStd::thread::hardware_concurrency() / cl_denominatorJobThreads);
```

Comment:
should reserve a core for the main thread and audio/system thread most likely

Reply:
The default now would be to create job threads to fill 1/2 the hardware threads, so there should be plenty left for the main and system threads. My expectation is that this ratio will go lower as time progresses.

Reply:
Yea I just meant that it may be worth having logic to ensure this is the case, just for the scenarios where we try to push this ratio higher.